### ISSUE TITLE
Login bypass

### ISSUE THREAD

**Issue by dfalling** [AppDoctorIo/accesspass#39]
Hi again!  I'm in the middle of implementing webauthn and was wondering if you're open to a contribution to facilitate it.

I was thinking something like this in Gatekeeper:
```elixir
  def bypass_log_in(username) do
    with {:ok, user} <- get_user(username),
         {:ok, token_body} <- overrides_mod().after_login(user) do
      {:ok,
       RefreshToken.add(
         user.user_id,
         token_body,
         refresh_expire_in()
       ) |> overrides_mod().login_return(user)}
    else
      {:error} -> {:error, "username is incorrect"}
      _ -> {:error, "error with bypass login endpoint"}
    end
end
```

**Comment by jpiepkow** [comment]
I have not got a chance to look into webauthn but if you want to put something together and PR, I am open to it.

**Comment by dfalling** [comment]
Great, thanks!  As you mentioned in #23, webauthn will bypass the typical username/password validation flow, so I'll simply be adding a way to login without providing a password.  I'll submit a WIP PR with just an implementation, and if that looks good will add tests and documentation.

**Comment by jpiepkow** [comment]
Looking forward to it!

**Comment by dfalling** [comment]
Closing this out, it was addressed in #40
